@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -665,17 +554,15 @@ export interface BasicExtrudedLineTechnique extends BaseTechnique, PolygonalTech
      * Style of both end caps. Possible values: `"None"`, `"Circle"`. A value of undefined maps to
      * `"Circle"`.
      */
-    caps?: string;
+    caps?: "None" | "Circle";
```

Comment:
Just as a general case, it would be a good moment to decide on letter casing policy for our themes, since we use ``value`` instead of ``Value`` in a lot of places, I would propose ``lowerCamelCase`` as our default.

I know this might make the PR more complex, so don't take this comment too strictly.